packages/cozy-client/src/CozyClient.js

```diff
@@ -103,9 +103,19 @@ class CozyClient {
         'Env used to instantiate CozyClient must have COZY_URL and COZY_CREDENTIALS'
       )
     }
+    let oauth
+    let token
+    try {
+      oauth = JSON.parse(COZY_CREDENTIALS)
+      token = JSON.parse(COZY_CREDENTIALS)
```

Comment:
Why do we have both ?

If parsing is successful, both oauth and token contain the same value, and I do not think the result is valable for the `token` option. Maybe you can remove the `token = JSON.parse` line ?

Can you add a test testing this behavior ?

Reply:
The logic should probably be from https://github.com/konnectors/libs/blob/db23f283acb10c2ed1651e07855495fe5396f583/packages/cozy-konnector-libs/src/libs/cozyclient.js#L14